boards/arm/rak4631_nrf52840/board.cmake

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2021 Guillaume Paquet <guillaume.paquet@smile.fr>
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(jlink "--device=nrf52" "--speed=4000")
+board_runner_args(pyocd "--target=nrf52840" "--frequency=4000000")
```

Comment:
@gpaquet85  can we add `board_runner_args(pyocd "--target=nrf52840" "--frequency=4000000")`?

Reply:
Yes it is as we have nRF52840 as µC and it is supported by pyocd ;)